iothub/service/src/DigitalTwin/DigitalTwinClient.cs

```diff
@@ -19,6 +19,7 @@ namespace Microsoft.Azure.Devices
 {
     /// <summary>
     /// The Digital Twins Service Client contains methods to retrieve and update digital twin information, and invoke commands on a digital twin device.
+    /// For more information, see <see href="https://github.com/Azure/azure-iot-sdk-csharp/blob/previews/RBAC/readme.md"/>
```

Comment:
I don't know of a way to make this not tied to a branch. We have to change this after we move it to the preview or master. What process does everyone follow here? The other option is I can keep it as preview as we will most definitely release a preview version and even if we don't update this value, it will be there in the preview branch. The feature branch can get deleted after the work is over. At the moment I just have it as a task on my card to do this update.

Reply:
Since the readme you are linking to should be the same between master and preview, I don't see why you couldn't make this link point towards the readme in the master branch, right?

Reply:
Makes sense. I could just update the readme in master itself.